pkg/mcs/resourcemanager/server/server.go

```diff
@@ -388,11 +395,8 @@ func (s *Server) startServer() (err error) {
 	if err != nil {
 		return err
 	}
-	s.listenURL, err = url.Parse(s.cfg.ListenAddr)
-	if err != nil {
-		return err
-	}
 	if tlsConfig != nil {
+		s.secure = true
```

Comment:
Do schedulering/server and tso/server need `s.secure = true`?

Reply:
I thnk yes

Reply:
I don't find out `s.secure = true` in scheduling/server, is it in another pr?